src/checks/jest/jest.ts

```diff
@@ -154,6 +161,16 @@ export const jestCheck = ({ data, sha, name = 'jest' }: JestInput): CheckRunComp
             path: relPath,
             raw_details: JSON.stringify(result, null, 2)
           }
+          // Try to keep the output small so we avoid truncation
+          if (JSON.stringify(annotation, null, 2).length > MAX_LINE_LENGTH) {
+            // Try to only remove raw details
+            annotation.raw_details = 'Too large to display'
```

Comment:
Would it be worth including some of the raw details instead of removing all the stack trace? 
Something like:
```suggestion
            annotation.raw_details = '[Too large to display all]'  + JSON.stringify(annotation, null, 2).substring(0, Math.floor(MAX_LINE_LENGTH / 2))
```

Reply:
I think the path, line number and title should be enough to figure out where to look. If we start to include significant parts of the truncated output, we need a lot more logic to ensure that we still stay under the limit. I think that is unnecessary as this logic is basically just made to not display failed mailman tests as pending/timed out.